docs/building-images.md

```diff
@@ -37,15 +37,17 @@ The operating system configuration section is entirely optional.
 The following describes the possible options for the operating system section:
 ```yaml
 operatingSystem:
-  installDevice: /path/to/disk
-  unattended: false
+  isoInstallation:
+    installDevice: /path/to/disk
+    unattended: false
   time:
     timezone: Europe/London
-    chronyPools:
-      - 1.pool.server.com
-    chronyServers:
-      - 10.0.0.1
-      - 10.0.0.2
+    ntp:
```

Comment:
Wasn't this supposed to still be `chrony`?

Reply:
We decided to go with the protocol instead of a specific implementation.